homeassistant/components/fyta/config_flow.py

```diff
@@ -0,0 +1,62 @@
+"""Config flow for FYTA integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from fyta_cli.fyta_connector import FytaConnector
+from fyta_cli.fyta_exceptions import (
+    FytaAuthentificationError,
+    FytaConnectionError,
+    FytaPasswordError,
+)
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+DATA_SCHEMA = vol.Schema(
+    {vol.Required(CONF_USERNAME): str, vol.Required(CONF_PASSWORD): str}
+)
+
+
+class FytaConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Fyta."""
+
+    VERSION = 1
+
+    async def async_step_user(
```

Comment:
The thing I'm now missing is the check for an already added entry. We can't use the unique id for this, but we can use async abort entries match. Please also make a test for this